### PR TITLE
Feat/add time to graph and network

### PR DESCRIPTION
**Pull request by sahand-asgarpour**
## Issue addressed
Solves #386 

## What has been done?
network.py is updated

### CODE REVIEW

File: hackathon/results_tasks_MS/combining_outputs_test/output1/static/output_graph/base_graph_hazard_edges.gpkg
Comment (Carsopre) [inline hackathon/results_tasks_MS/combining_outputs_test/output1/static/output_graph/base_graph_hazard_edges.gpkg:1]: You should not be committing these files as they belong to the hackathon.

File: hackathon/results_tasks_MS/combining_outputs_test/output2/static/output_graph/base_graph_hazard_edges.gpkg
Comment (Carsopre) [inline hackathon/results_tasks_MS/combining_outputs_test/output2/static/output_graph/base_graph_hazard_edges.gpkg:1]: You should not be committing these files as they belong the hackathon.

File: ra2ce/network/networks.py [hunk 1]
```diff
@@ -154,6 +156,26 @@ def _export_network_files(
         self.graph_files.set_graph(graph_type, network)
 
     def _get_new_network_and_graph(self, export_types: list[str]) -> None:
+        def _get_edges_time_hours(_graph: nx.MultiGraph) -> dict:
```
Comment (Carsopre) [inline ra2ce/network/networks.py:159]: You are assuming that `avgspeed` will be present for all those dataframes. This makes the whole test bench fail. Please modify the code below so it avoids it.

**Review by Carsopre — CHANGES_REQUESTED** [review]

File: ra2ce/network/networks.py (hunk 1 above)
Comment (sahand-asgarpour) [reply ra2ce/network/networks.py:159]: I checked and here e[-1].get("avgspeed") returns None if avgspeed does not exist. I tested it. Where does the issue come from?

**Review by Carsopre — APPROVED** [review]

**Pull request merged**